tcp/tcp.py

```diff
@@ -103,8 +112,26 @@ def __process_requests(self, client_socket, client_addr):
             req = fd.readline()
             while req:
                 req = req.strip()
-                logger.info("TCP - recieved request {}".format(req))
-                if req == RECV_IS_UNINSTALLED:
+                logger.info("TCP - received request {}".format(req))
+                if req == RECV_RECEIVED_DATE:
```

Comment:
I like to use [dispatch dictionaries](https://alysivji.github.io/quick-hit-dictionary-dispatch.html) when I need to write big if statements.